microk8s-resources/actions/enable.metallb.sh

```diff
@@ -14,6 +14,15 @@ fi
 
 echo "Enabling MetalLB"
 
+ALLOWESCALATION=false
```

Comment:
Note that unlike the [coredns fix](https://github.com/ubuntu/microk8s/pull/551), quotes are not used here or on line 20.

I initially tried with quotes, but when I installed the snap and tried to install, I got the following error message:
```
Applying registry manifest
namespace/metallb-system unchanged
error: error validating "STDIN": error validating data: ValidationError(PodSecurityPolicy.spec.allowPrivilegeEscalation): invalid type for io.k8s.api.policy.v1beta1.PodSecurityPolicySpec.allowPrivilegeEscalation: got "string", expected "boolean"; if you choose to ignore these errors, turn validation off with --validate=false
Failed to enable metallb
```

That made me think the same issue might apply to the coredns fix linked above,  but I tested several times and the coredns fix seems fine as-is.